frontend/ui/data/optionsutil.lua

```diff
@@ -0,0 +1,133 @@
+--[[--
+This module contains miscellaneous helper functions for the creoptions and koptoptions.
+]]
+
+local Device = require("device")
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+local Screen = Device.screen
+local T = require("ffi/util").template
+
+local optionutil = {}
```

Comment:
While it doesn't really matter for the code itself, I can't help but notice that the file and the use say option*s*util and in this file it all says optionutil. optionsutil probably sounds better. ;-)